src/js/BookReader.js

```diff
@@ -581,6 +581,21 @@ BookReader.prototype.drawLeafs = function() {
   }
 };
 
+BookReader.prototype.leafTemplate = function(styles, index) {
```

Comment:
Also, let's rename this to be whater the css class is; i.e. `createPageContainer`